src/services/namex.services.ts

```diff
@@ -139,9 +139,8 @@ export default class NamexServices {
         })
         sessionStorage.removeItem('checkedOutBy')
         sessionStorage.removeItem('checkedOutDt')
-
-        return true
       }
+      return true
```

Comment:
Fix for no return value (which is handled as false) when the NR is not checked out.